avail/src/main/resources/VmVersionTemplate.txt

```diff
@@ -1,21 +1,21 @@
 /*
- * settings.gradle.kts
- * Copyright © 1993-2022, The Avail Foundation, LLC.
+ * VmVersion.kt
```

Comment:
Wrong file name.

Reply:
@markATavail This is the correct name to put here. This is a file that contains the template to code generate `VmVersion.kt`. So the name here is correct.